salt/pillar/libvirt.py

```diff
@@ -91,8 +99,10 @@ def gen_hyper_keys(minion_id,
         with salt.utils.fopen(srvinfo, 'w+') as fp_:
             infodat = ('organization = salted\ncn = {0}\ntls_www_server'
                        '\nencryption_key\nsigning_key'
-                       '\ndigitalSignature').format(
-                               __grains__['fqdn'])
+                       '\ndigitalSignature\n'
+                       'expiration_days = {1}'
+                       ).format(
+                               __grains__['fqdn'], expiration_days)
```

Comment:
This line can be on the previous though. But maybe multi-line template would work better here?